### PR TITLE
Pimax Module Candidate

### PR DESCRIPTION
**Pull request by dfgHiatus**

### CODE REVIEW

**Comment by regzo2** [comment]
Since these docs are providing a very similar goal to the Varjo docs candidates for the Pimax module, the info from this [comment ](https://github.com/VRCFaceTracking/docs/pull/3#issuecomment-1510217666)would be relevant to these docs as well

**Review by benaclejames — CHANGES_REQUESTED** [review]
Looks good, just be sure use the correct file paths. Getting a lot of `[WARNING] Docs markdown link couldn't be resolved: (./modules.mdx) in "C:\Users\benac\Documents\docs\docs\tutorial-modules\pimax.mdx" for version current` errors when trying to run locally

File: docs/tutorial-modules/pimax.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Assuming there are no connection issues with the hardware, the connection hardware components should be initialized by VRCFT after a few seconds.
+
+### Calibrating Eye Tracking
+
+There is no way to calibrate this module. 
+
+## Uninstalling The VRCFTPimaxModule
+
+To uninstall The VRCFTPimaxModule, navigate to `%\AppData\Roaming\VRCFaceTracking\CustomLibs` and delete the file called VRCFTPimaxModule.dll.
```
Comment (benaclejames) [inline docs/tutorial-modules/pimax.mdx:47]: Incorrectly formatted path

File: docs/tutorial-modules/pimax.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The VRCFTPimaxModule contains the runtime used to interface with Pimax eye tracking hardware and software on Windows PCs.
+
+## Set-up
+
+Setup of the Pimax Module is fairly simple, though can be a little confusing depending on your hardware/software configuration. Please read carefully to avoid any issues. 
+
+:::note
+This guide assumes you have PiTool and ASeeVR installed and updated prior to using the VRCFTPimaxModule. 
+If you do not have the [PiTool](https://pimaxcom.myshopify.com/pages/pitool-download) and/or [ASeeVR](https://drive.google.com/file/d/1ELDtOnMa-MkgchmWFf7w5an-iPOFtQL8/view?ref=Nebo&_ga=2.157958394.1121053981.1651257342-354291550.1593067405)
```
Comment (benaclejames) [inline docs/tutorial-modules/pimax.mdx:13]: The ASeeVR link denies public access to download

File: docs/tutorial-modules/pimax.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| Droolon Pi1                         | `✅` Yes | `❌` No  |
+
+## Using The Pimax Module
+
+To use your Pimax's eye tracking with VRChat, you will need to use the [The Pimax VRCFT Module](https://github.com/benaclejames/VRCFTPimaxModule) with the VRCFaceTracking software.
+For more information on modules and how to use them, see [Modules](./modules.mdx).
+
+Assuming there are no connection issues with the hardware, the connection hardware components should be initialized by VRCFT after a few seconds.
+
+### Calibrating Eye Tracking
```
Comment (benaclejames) [inline docs/tutorial-modules/pimax.mdx:41]: If there's no way to calibrate, why mention it in the first place?

Comment (dfgHiatus) [reply docs/tutorial-modules/pimax.mdx]: Resolved in #08c40db. Removed this section

File: docs/tutorial-modules/pimax.mdx (hunk 2 above)
Comment (dfgHiatus) [reply docs/tutorial-modules/pimax.mdx]: Resolved in #08c40db. Changed this link to Guppy's Github

File: docs/tutorial-modules/pimax.mdx (hunk 1 above)
Comment (dfgHiatus) [reply docs/tutorial-modules/pimax.mdx]: Resolved in #08c40db.

Comment (benaclejames) [reply docs/tutorial-modules/pimax.mdx]: This appears to still be an issue in latest

**Review by benaclejames — APPROVED** [review]
Looking good! 👍

**Pull request merged**